docs/ENV.md

```diff
@@ -57,3 +57,10 @@ Name of the Google Analytics Property corresponding to the Tracking Code mention
 ### Mixpanel
 #### `VUE_APP_MIXPANEL_PROJECT_TOKEN`
 This is the project token for your mixpanel project which you can view in the Mixpanel Dashboard.
+
+### Chatwoot
```

Comment:
would be good to add one saying what is Chatwoot used for.
I think would be good to do the same under the `Mixpanel` section too.